infra/integration-image/Dockerfile.downstream

```diff
@@ -19,7 +19,7 @@ RUN curl -L https://github.com/drone/drone-cli/releases/download/v0.8.5/drone_li
     install drone /usr/bin
 
 RUN echo '#!/bin/bash' >> /usr/bin/trigger
-RUN echo 'num=$(drone build ls --format "{{.Number}} {{.Status}}" "$DOWNSTREAM_REPO" | grep -v running | head -n1 | cut -d" " -f1)' >> /usr/bin/trigger
+RUN echo 'num=$(drone build ls --format "{{.Number}} {{.Status}}" --event push --branch "$DOWNSTREAM_BRANCH" "$DOWNSTREAM_REPO" | grep -v running | head -n1 | cut -d" " -f1)' >> /usr/bin/trigger
```

Comment:
should this include `tag` as an event or is it deliberately filtering those out?

Reply:
`tag` is deliberately omitted; we don't want to run `tag` builds twice to avoid confusion ("Which `v1.2.3-milestone` build is the real one"?)